project/Helpers.scala

```diff
@@ -94,33 +94,29 @@ object C {
         run / fork := false,
         Global / cancelable := false, // ctrl-c
         libraryDependencies ++= Seq(
-          "com.typesafe.akka" %% "akka-http" % V.akkaHttp,
-          "com.typesafe.akka" %% "akka-http2-support" % V.akkaHttp ,
-          "com.typesafe.akka" %% "akka-http-spray-json" % V.akkaHttp,
+          "com.typesafe.akka" %% "akka-actor" % V.akka,
```

Comment:
This is redundant with akka-actor-typed. We want to be modern and typed actors are more modern than the untyped one (deprecated).

Reply:
removed